_gsocproposals/2019/proposal_ActsAlign.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Improving the precision of algorithms to hunt particles
+layout: gsoc_proposal
+project: Acts, IRIS-HEP
```

Comment:
Currently, proposals can only belong to a single project, could you please choose only one ? (Given the description, I would suggest to keep Acts)

Reply:
I went with Acts, but this is tricky because my work (and Xiaocong's) on Acts is through IRIS-HEP.